deploy/charts/cert-manager/templates/webhook-service.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/component: "webhook"
     {{- include "labels" . | nindent 4 }}
+{{- if .Values.serviceLabels }}
```

Comment:
Instead of using the `serviceLabels` value from the controller's values block, can we create a new `serviceLabels` value [in the webhook section](https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L202-L325)?